### PR TITLE
fix typos in documents

### PR DESCRIPTION
**Pull request by i110**
I applied a spell checker (aspell) to all document htmls, and fix typos.

### CODE REVIEW

File: doc/benchmarks.html [hunk 1]
```diff
@@ -65,7 +65,7 @@ <h3 id="download-timings">Download Timings</h3>
...
+The chart below compares the first-paint times and download completion times of different web browsers / HTTP servers on a simulated network of 8Mbps bandwidth with 100ms latency, which is typical for today's mobile networks<sup><a href="#note_2" id="#cite_2" title="A fork of http2rulez.com was used as the target website; bandwidth and latency were induced to local network using qdisc, specifically by running tc qdisc replace dev eth1 root handle 1:0 tbf rate 8192kbit burst 2048 latency 100ms; sudo tc qdisc add dev eth1 parent 1:1 netem delay 100ms, and sysctl -w net.ipv4.tcp_no_metrics_save=1.">2</sup></a></sup>.
```
Comment (i110) [inline doc/benchmarks.html]: `s/typcial/typical/`

File: doc/configure/file_directives.html [hunk 2]
```diff
@@ -213,7 +213,7 @@
 </p>
 
 <p>
-The sequence of filenames afer search from left to right, and the first file that existed is sent to the client.
+The sequence of filenames are searched from left to right, and the first file that existed is sent to the client.
```
Comment (i110) [inline doc/configure/file_directives.html]: Is this correct?

File: srcdoc/benchmarks.mt [hunk 3]
```diff
@@ -10,7 +10,7 @@ Providing quick response to user is more important than anything else in web per
...
+The chart below compares the first-paint times and download completion times of different web browsers / HTTP servers on a simulated network of 8Mbps bandwidth with 100ms latency, which is typical for today's mobile networks<?= $note->(q{<a href="https://github.com/kazuho/http2rulez.com">A fork of http2rulez.com</a> was used as the target website; bandwidth and latency were induced to local network using <a href="http://linux-ip.net/articles/Traffic-Control-HOWTO/components.html">qdisc</a>, specifically by running <code>tc qdisc replace dev eth1 root handle 1:0 tbf rate 8192kbit burst 2048 latency 100ms; sudo tc qdisc add dev eth1 parent 1:1 netem delay 100ms</code>, and <code>sysctl -w net.ipv4.tcp_no_metrics_save=1</code>.}) ?>.
```
Comment (i110) [inline srcdoc/benchmarks.mt]: `s/typcial/typical/`

**Comment by kazuho** [comment]
Thank you for the fixes!

File: doc/configure/file_directives.html (hunk 2 above)
Comment (kazuho) [inline doc/configure/file_directives.html]: Yes!

**Comment by kazuho** [comment]
@i110 How was it like to apply aspell?

I mean if the output is easy enough to find spelling errors, it might be a good idea to add the spell-check script to `misc` or even as an author test.

**Comment by i110** [comment]
@kazuho 
I applied this command:

```
find ./doc -name "*html" | ruby -r nokogiri -ne 'BEGIN{$c=[]};$c<<$_.chomp;END{puts Nokogiri::HTML($c.join(" ")).search("//text()")}' $@ | aspell --lang=en --ignore-case list | sort -u
```

But, to get this to work good, you have to get 秘伝のタレ (a dictionary which contains many technical or H2O specific words). My dictionary is very adhoc and imperfect at all, so I don't think this works well everytime.

My current dictionary is here:
https://gist.github.com/i110/d2c7d51f8459cbeefc18133c28bb536f

**Pull request merged**